### PR TITLE
Branding changes

### PR DESCRIPTION
**Pull request by jr-projectn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,30 +12,23 @@ The minimum supported version of Python is version 3.
...
-aws [--profile PROFILE] configure set bolt_custom_domain <bolt custom domain>
-aws [--profile PROFILE] configure set bolt_az <preferred availability zone ID>
+aws configure set plugins.bolt awscli-plugin-granica
+aws [--profile PROFILE] configure set granica_custom_domain <bolt custom domain>
+aws [--profile PROFILE] configure set granica_az <preferred availability zone ID>
 ```
-`bolt_custom_domain`
+`granica_custom_domain`
 
 For example, to activate a Bolt service using an internal load balancer hosted at `https://bolt.us-east-1.project.n` for the `staging` AWS profile, run:
```
Comment (tarangv) [inline README.md:28]: Is the url still .project.n ?

Comment (tarangv) [reply README.md:28]: Also, Bolt service is referred above

Comment (jr-projectn) [reply README.md]: Updated

File: awscli-plugin-granica/bolt_cli_hooks.py [hunk 2]
```diff
@@ -34,10 +34,10 @@ def _activate(parsed_args, **kwargs):
         except Exception as e:
             pass
 
-    if 'bolt_custom_domain' in profile and region is not None:
+    if 'granica_custom_domain' in profile and region is not None:
```
Comment (tarangv) [inline awscli-plugin-granica/bolt_cli_hooks.py:37]: Change file names bolt_router.py and bolt_cli_hooks.py ?

File: awscli-plugin-granica/bolt_cli_hooks.py [hunk 3]
```diff
@@ -46,8 +46,8 @@ def _activate(parsed_args, **kwargs):
         return
 
     az_id = None
-    if 'bolt_az' in profile:
-        az_id = profile['bolt_az']
+    if 'granica_az' in profile:
+        az_id = profile['granica_az']
     else:
         try:
             az_id = get_availability_zone_id()
```
Comment (tarangv) [inline awscli-plugin-granica/bolt_cli_hooks.py:53]: how about _global_bolt_router code below:

File: installer.py [hunk 4]
```diff
@@ -64,7 +64,7 @@ def run(region, domain, account_id, install_server):
       'prompt':"Installing Plugin...",
       'commands':[
         'pip3 install --user .',
-        'aws configure set plugins.bolt awscli-plugin-bolt'
+        'aws configure set plugins.granica awscli-plugin-granica'
       ]
     },
```
Comment (tarangv) [inline installer.py:70]: I see some command below:

aws --profile={} configure set bolt.url https://bolt.{}'.format(region,domain

Comment (tarangv) [reply installer.py:70]: Does it become quicksilver.

File: setup.py [hunk 5]
```diff
@@ -5,10 +5,10 @@
...
-    name='awscli-plugin-bolt',
-    packages=['awscli-plugin-bolt'],
+    name='awscli-plugin-granica',
+    packages=['awscli-plugin-granica'],
     version='2.0.4',
-    description='Bolt plugin for AWS CLI',
+    description='Granica plugin for AWS CLI',
     long_description=open('README.md').read(),
     long_description_content_type='text/markdown',
     author='Project N',
```
Comment (tarangv) [inline setup.py:14]: I see url=https://gitlab.com/projectn-oss/projectn-bolt-awscli" below

perhaps we keep this folder/repo protected, and create a new

https://gitlab.com/projectn-oss/projectn-granica-awscli, with all the changes we are doing here?

File: installer.py (hunk 4 above)
Comment (tarangv) [reply installer.py:70]: Does it become quicksilver.

File: awscli-plugin-granica/bolt_cli_hooks.py (hunk 2 above)
Comment (jr-projectn) [reply awscli-plugin-granica/bolt_cli_hooks.py:37]: They are not visible to the Cx, trying to keep the scope narrow

File: awscli-plugin-granica/bolt_cli_hooks.py [hunk 6]
```diff
@@ -34,10 +34,10 @@ def _activate(parsed_args, **kwargs):
...
             pass
 
-    if 'bolt_custom_domain' in profile and region is not None:
+    if 'granica_custom_domain' in profile and region is not None:
         scheme = 'https' 
-        service_url = f"quicksilver.{region}.{profile['bolt_custom_domain']}"
-        hostname = f"bolt.{region}.{profile['bolt_custom_domain']}"
+        service_url = f"quicksilver.{region}.{profile['granica_custom_domain']}"
+        hostname = f"bolt.{region}.{profile['granica_custom_domain']}"
     elif 'bolt_hostname' in profile and 'bolt_url' in profile:
```
[the remark this answers is not in the document]
Comment (jr-projectn) [reply awscli-plugin-granica/bolt_cli_hooks.py:41]: This var is not needed

File: installer.py (hunk 4 above)
Comment (jr-projectn) [reply installer.py:70]: This variable is not needed.

**Pull request merged**